Packages/com.unity.inputsystem/InputSystem/InputSystem.cs

```diff
@@ -3727,30 +3733,15 @@ internal static void TrackDirtyInputActionAsset(InputActionAsset asset)
         }
 
 #else
-        private static void InitializeInPlayer(IInputRuntime runtime = null, InputSettings settings = null, InputActionAsset actions = null)
+        private static void InitializeInPlayer(IInputRuntime runtime = null, InputSettings settings = null)
```

Comment:
Unfortunately, this (and subsequent changes in this file) are going to conflict with my FEPM branch. However, the changes aren't too complicated, and I should be able to resolve the conflicts without too much trouble.

Reply:
Lets sort it out together on that branch. I definitely think there is room for improvement around this logic though, at least from a readability and debugging perspective - outside of this PR.